models/Translation/Dao.php

```diff
@@ -101,11 +101,16 @@ public function save(): void
                         continue;
                     }
 
+                    if ($text != strip_tags($text)) {
+                        $text = $sanitizer->sanitizeFor('body', $text);
+                        $this->model->addTranslation($language, $text);
```

Comment:
please see https://github.com/pimcore/admin-ui-classic-bundle/pull/309 removed it from admin controller so sanitize happens only once